src/attacks.ts

```diff
@@ -24,7 +24,7 @@ function computeRange(square: Square, deltas: number[]): SquareSet {
   let range = SquareSet.empty();
   for (const delta of deltas) {
     const sq = square + delta;
-    if (isValid(sq) && Math.abs(squareFile(square) - squareFile(sq)) <= 2) {
+    if (isValid(sq) && Math.abs(squareFile('chess')(square) - squareFile('chess')(sq)) <= 2) {
```

Comment:
so why do we want to default to `chess` in a lot of these positions?

Reply:
This is the move generation code. As I noted in our meeting, this is still chess only.